rero_ils/modules/ill_requests/models.py

```diff
@@ -33,3 +33,12 @@ class ILLRequestIdentifier(RecordIdentifier):
         db.BigInteger().with_variant(db.Integer, 'sqlite'),
         primary_key=True, autoincrement=True,
     )
+
+
+class ILLRequestStatus:
+    """Available status for an ILL request."""
+
+    PENDING = 'pending'
```

Comment:
Perhaps you need translations here...

Reply:
According to other Status classes (`ItemStatus` and `ItemIssueStatus`) it seems not necessary because we don't use them as user message part.